http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20210217_add_app_pvc_id.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="add_app_pvc_id">
+        <addColumn tableName="APP">
+            <column name="galaxyPvcId" type="varchar(254)">
```

Comment:
This is probably fine for now, but we should think about data model for `CUSTOM` app type. Maybe we want some different tables containing Galaxy-specific fields.

Reply:
I see...we can probably wait until Justin's PR merged and revisit